src/test/java/org/jabref/architecture/MainArchitectureTests.java

```diff
@@ -104,8 +105,7 @@ public void firstPackageIsIndependentOfSecondPackage(String firstPackage, String
                     })
                     .collect(Collectors.toList());
 
-            Assert.assertEquals("The following classes are not allowed to depend on " + secondPackage,
-                    Collections.emptyList(), files);
+            Assertions.assertEquals(Collections.emptyList(), files, "The following classes are not allowed to depend on " + secondPackage);
```

Comment:
I would prefer a static import of asset und

Reply:
Done!